### DISCUSSION TITLE
Umi4 是移除了 'addUmiExports' 的API方法了吗? 后续如何演进?

### DISCUSSION THREAD

**Discussion by HaishengLiang** [umijs/umi#8363] [Q&A]
之前的业务使用了该方案,  想了解后续的设计

**Comment by fz6m** [comment]
把导出写在插件的 `index.ts` 或者 `index.tsx` 就可以了：

```ts
./plugin-${name}
└── index.ts
```

他会自动被 `.umi/exports.ts` 导出，也就是 `umi` 的 alias 。从而可以从 `umi` 中 import 。

FYI：https://github.com/umijs/umi/blob/97f0f73bc17de0d3ffeb8c217711d0870614fe33/packages/plugins/src/locale.ts#L237-L244

**Reply by HaishengLiang** [reply]
我想了解这种方式被umi扫描的范围, 是在.umi目录下的文件都会被自动导出吗?

**Reply by fz6m** [reply]
自动导出的范围：每个插件下的 `index.ts` or `index.tsx` 文件

**Reply by topwood** [reply]
@fz6m 现在看起来上面例子中没有Index.ts了
